modules/@angular/docs/cheatsheet/component-configuration.md

```diff
@@ -8,6 +8,12 @@ so the `ng.core.Directive` configuration applies to components as well{@endtarge
 so the `@Directive` configuration applies to components as well{@endtarget}
 
 @cheatsheetItem
+syntax(ts js):
```

Comment:
Did I understand this correctly? I set `syntax(ts js)` because the `moduleId` property applies to `ts` and `js`, but not to dart.